MAUI/LinearGauge/getting-started.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: post
+title: Getting started with .NET MAUI Linear Gauge control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI Linear Gauge (SfLinearGauge) control, its elements, and more. 
+platform: maui
+control: SfLinearGauge
+documentation: ug
+---
+
+# Getting Started with .NET MAUI Linear Gauge
+
+This section explains the steps required to add the [`.NET MAUI Linear Gauge`]() control and its elements such as axis, range, and pointer. This section covers only basic features needed to get started with Syncfusion linear gauge control.
+
+## Creating an application using the .NET MAUI Linear Gauge
+
+* Create a new .NET MAUI application in the Visual Studio.
+
+* Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add SfLinearGauge to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Gauges] and then install it.
+
+* Import the control namespace `Syncfusion.Maui.Gauges` in XAML or C# code.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+xmlns:gauge="clr-namespace:Syncfusion.Maui.Gauges;assembly=Syncfusion.Maui.Gauges"
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+using Syncfusion.Maui.Gauges;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+* Initialize the SfLinearGauge control
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<gauge:SfLinearGauge />
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearGauge gauge = new SfLinearGauge();
+this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Register the handler
+
+Syncfusion.Maui.Core nuget is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file register the handler for Syncfusion core.
```

Comment:
Syncfusion.Maui.Core NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.